client/dataEntry.js

```diff
@@ -31,20 +31,12 @@ Template.dataEntry.helpers({
   },
 
   isSelectedRoundClosed: function() {
-    if(!this.roundId) {
-      // If there's no round selected, then the selected round is definitely
-      // *not* closed =)
-      return false;
-    }
-    return Rounds.findOne(this.roundId).isClosed();
+    round = Rounds.findOne(this.roundId);
```

Comment:
Maybe this is a philosophy question.

I think you build sane systems by validating data as it enters your system. Once it's inside you **trust** it. "*Let's double check all data each time we touch it*" is a serious anti pattern.

I think this `roundId` comes out of our DB, and therefore we should trust it.

Or maybe I'm missing the point here.